packages/web-features/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "target": "ES2016",
     "module": "ESNext",
-    "moduleResolution": "Bundler"
+    "moduleResolution": "Bundler",
+    "skipLibCheck": true
```

Comment:
Needed to add this because it was throwing a build error on build due to an issue with the `markdown-it` module. Adding this tells it not to get cranky if some of the node modules aren't easily compatible with ESM

Error was:

```
../../node_modules/@types/markdown-it/dist/index.cjs.d.ts:981:23 - error TS2709: Cannot use namespace 'LinkifyIt' as a type
```

Reply:
This is a very blunt instrument to accommodate Eleventy's lack of ESM support—I'm not comfortable with it. I think a better way to fix this would be:

- Move `gh-pages` to `/gh-pages/src`
- Move all the root Eleventy config to `/gh-pages` (i.e., `./eleventy.config.js` → `gh-pages/eleventy.config.js`, move the `scripts` changes from `package.json` into `gh-pages/package.json`)

Then we can run gh-pages-related things separate from everything else, instead of relaxing the type checks on the code that generates the web-features data.

Reply:
Done